src/injectScripts.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
-
 const fs = require('fs');
 const posthtml = require('posthtml');
 
+const DOCTYPE = '!DOCTYPE';
+const HEAD = 'head';
+const HTML = 'html';
 const SCRIPT = 'script';
+const TITLE = 'title';
+
+const JS_TEST = 'js-test.js';
 const TEST_HARNESS = 'testharness.js';
 const TEST_HARNESS_REPORT = 'testharnessreport.js';
-const JS_TEST = 'js-test.js';
+
+const INDENTATION_REGEX = new RegExp('^\n[ ]*$');
```

Comment:
Might as well use `\s` to capture all whitespaces including tabs (yeah there are some...)?

Reply:
Replacing with /^\n\s*$/g